client/src/components/DeckCreateForm.js

```diff
@@ -14,6 +14,7 @@ const CREATE_DECK_MUTATION = gql`
 `;
 
 class DeckCreateForm extends Component {
+  
```

Comment:
When making a PR, try avoiding committing unnecessary formatting changes. It blows up the diffs and creates noise in the git history. It also makes later git annotates or git blames more difficult (When trying to find out who is responsible for a line of code)